code/modules/vehicles/mecha/mecha_actions.dm

```diff
@@ -34,6 +34,12 @@
 	if(!owner || !chassis || !(owner in chassis.occupants))
 		return
 
+	if(!chassis.internal_tank) //Just in case.
+		chassis.use_internal_tank = FALSE
+		chassis.balloon_alert(owner, "No tank available!")
```

Comment:
balloon alerts are not supposed to be capitalized

Reply:
Changed.